Simple_Prep_docker

```diff
@@ -66,11 +66,9 @@ sed-i "$dockerfile" -e '/DL_APT/d'
 tag_=simple_prep_${USER}_${DL_DIST}_${DL_DATE}
 tag=repronim:${tag_}
 echo "I: tag $tag"
-if docker images | grep -q repronim.*${tag_}; then
-    echo "I: tag $tag already exists -- skipping rebuilding"
-else
+if [ $(command -v docker) ]; then
     docker build -t $tag -f $dockerfile .
```

Comment:
it should rebuild based on content of dockerfile and it's connected files and not tag. tag doesn't guarantee much if we have complicated dependencies.